client/extensions/woocommerce/app/products/product-form-variations-card.js

```diff
@@ -54,13 +50,16 @@ export default class ProductFormVariationCard extends Component {
 				icon=""
 				expanded={ true }
 				className="products__variation-card"
-				header={ ( <FormToggle onChange={ this.handleToggle } checked={ this.state.isVariation }>
+				header={ ( <FormToggle onChange={ this.handleToggle } checked={ 'variable' === product.type }>
 					{ variationToggleDescription }
 				</FormToggle>
 				) }
 			>
-				{ this.state.isVariableProduct && (
-					<ProductVariationTypesForm />
+				{ 'variable' === product.type && (
+					<ProductVariationTypesForm
+						product={ product }
+						editProductAttribute={ this.props.editProductAttribute }
```

Comment:
Why not add `editProductAttribute` to the destructuring above to keep the code here as brief as possible?